SmartDeviceLink/SDLSystemCapabilityManager.m

```diff
@@ -184,18 +193,20 @@ - (void)sdl_registerResponse:(SDLRPCResponseNotification *)notification {
     self.pcmStreamCapability = response.pcmStreamCapabilities;
 }
 
-
-
-
 /**
  *  Called when a `SetDisplayLayoutResponse` response is received from Core. If the template was set successfully, the the new capabilities for the template are saved.
  *
  *  @param notification The `SetDisplayLayoutResponse` response received from Core
  */
 - (void)sdl_displayLayoutResponse:(SDLRPCResponseNotification *)notification {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
     SDLSetDisplayLayoutResponse *response = (SDLSetDisplayLayoutResponse *)notification.response;
+#pragma clang diagnostic pop
     if (!response.success.boolValue) { return; }
 
+    self.displays = [self sdl_createDisplayCapabilityListFromSetDisplayLayoutResponse:response];
```

Comment:
This would be conflicting to what was accepted with the proposal:
> If the application is connected to a < 6.0 or if the head unit did not provide DisplayCapability or WindowCapability for the DEFAULT_WINDOW yet, the system capability manager should convert DisplayCapabilities, ButtonCapabilities, SoftButtonCapabilities and PresetBankCapabilities objects and the string of displayName from RegisterAppInterfaceResponse and SetDisplayLayoutResponse into a new DisplayCapability object. This object should be stored in the DISPLAYS enum value.

Reply:
Sorry, I need to clarify. I mean that if you're connected to a 6.0+ head unit, then the `displays` data should not be overwritten by the deprecated data. On a < 6.0 head unit, as you quote, then it should pull the deprecated data. Currently, the code overwrites the `OnSystemCapabilityUpdated.displayCapability` data with the deprecated `SetDisplayLayoutResponse` data.

Reply:
That should work